native_implemented/otp/src/erlang/encode_unsigned_1.rs

```diff
@@ -0,0 +1,58 @@
+#[cfg(all(not(target_arch = "wasm32"), test))]
+mod test;
```

Comment:
We're moving away from unit tests and preferring writing integration tests in Erlang now under [`native_implemented/otp/tests/internal/lib`](https://github.com/lumen/lumen/blob/develop/native_implemented/otp/tests/internal/lib)

Reply:
OK, does that also mean that we're moving away from proptest as well?

How do I run these integration tests in local dev?

Reply:
Never mind, for now. I think I figured out how to wire up and run the erlang integration tests.